gf-webapp/src/logic/rooms/roomsPage/roomBlock/roomItem/description/Description.tsx

```diff
@@ -35,10 +38,15 @@ export const Description = (props: DescriptionProps) => {
     >
       {// TODO: Title shouldn't have props. Task # 121
       }
-      <Title roomNumber={props.roomNumber} />
+      <Title
+        roomNumber={props.roomNumber}
+        type={props.type}
```

Comment:
It's wrong. Title is not equal type. Title shouldn't depends on type of room. We have 3 types of room but hotel can have 100 rooms with different titles.